variants/RUMBA32_F446VE/PeripheralPins.c

```diff
@@ -0,0 +1,337 @@
+/*
```

Comment:
You could update it to add USB pin mapping:
https://github.com/stm32duino/Arduino_Tools/blob/master/src/genpinmap/Arduino/STM32F446V(C-E)Tx/PeripheralPins.c

Reply:
Done for USB FS (no HS on board).